### PR TITLE
Fix/fixes issue #45 - adds proxy settings for retrieving tweets with proxy

### LINKED ISSUE #45: Tweets don't show if user is using proxy
**What is the expected behavior?**
Tweets show up if the user has a proxy setup

**What is the actual behavior?**
Tweets show up correctly

**What steps can be taken to replicate the problem?**
Set a display to pro
Add Twitter to a Presentation and connect your account
Schedule Twitter to the display
Setup the display to [use proxy](https://help.risevision.com/hc/en-us/articles/115003749183-How-do-I-configure-Rise-Player-to-use-a-proxy-)
Tweets don't show up

### PR DESCRIPTION
**Pull request by jungeunyoon**
Closes #45

### CODE REVIEW

File: src/config/config.js [hunk 1]
```diff
@@ -37,6 +37,11 @@ function isAuthorized() {
   return authorized;
 }
 
+function getProxyUri() {
+    const proxy = common.getProxyAgents();
+    return (proxy.httpsAgent && proxy.httpsAgent.proxyUri) || (proxy.httpAgent && proxy.httpAgent.proxyUri) || null // eslint-disable-line no-extra-parens
+}
+
```
Comment (santiagonoguez) [inline src/config/config.js]: I see this also in:

https://github.com/Rise-Vision/local-storage-module/blob/master/src/files/proxy.js

Shouldn't it be added to common instead so we can reuse it ?

Comment (jungeunyoon) [reply src/config/config.js]: @santiagonoguez Fair point. I can go ahead and add it to common but will Delivery be covering the changes needed for LS?

Comment (santiagonoguez) [reply src/config/config.js]: @jungeunyoon I'll change local storage while I'm still here :)

File: src/api/twitter-wrapper.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const Twitter = require("twitter");
 const config = require("../../src/config/config");
+const commonConfig = require("common-display-module");
```
Comment (santiagonoguez) [inline src/api/twitter-wrapper.js]: I guess this should now point to proxy.js

**Comment by jungeunyoon** [comment]
@Rise-Vision/content Please hold off on reviewing. Need to modify tests.

**Pull request closed without merging**